### PR TITLE
Gabbi/url encode

### LINKED ISSUE #504: Make sure to URL encode all keys
This applies to singleton put and get operations. Without this, puts and gets with keys that have spaces will fail.

### PR DESCRIPTION
**Pull request by gabbifish**
This PR closes #504. Since this change also impacts how KV command methods import the `url_encode_key()` function from the mod.rs for the kv package, it also implements the change suggested here: https://github.com/cloudflare/wrangler/pull/405#discussion_r318173571

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -120,7 +120,7 @@ fn run() -> Result<(), failure::Error> {
                         )
                         .arg(
                             Arg::with_name("expiration-ttl")
-                            .help("Number of seconds for which the entries should be visible before they expire. At least 60. Takes precedence over 'expiration' option")
+                            .help("Number of seconds for which the entries should be visible before they expire. At least 60")
```
Comment (ashleymichal) [inline src/main.rs]: did you mean to remove this?

Comment (gabbifish) [reply src/main.rs]: Oop! That is an accident. Fixing.

Comment (gabbifish) [inline src/main.rs]:
```suggestion
                            .help("Number of seconds for which the entries should be visible before they expire. At least 60. Takes precedence over 'expiration' option")
```

Comment (EverlastingBugstopper) [inline src/main.rs]: `. At least 60.` --> `. Must be at least 60 seconds.`

**Pull request merged**